controller/webhook.go

```diff
@@ -294,7 +302,7 @@ func extractParameters(host string, name string, region string) (string, string,
 		region = hostModified[:strings.IndexByte(hostModified, '.')]
 	}
 
-	return host, name, region
+	return host, name, region, unsignedPayload
```

Comment:
Can we convert `unsignedPayload` to a boolean in this function and return the boolean directly?